.github/workflows/publish-cli.yml

```diff
@@ -0,0 +1,78 @@
+name: "publish-cli"
+
+# Only a single job with this concurrency can run at any given time
+concurrency: publish-cli
+
+on:
+  workflow_dispatch:
```

Comment:
@gerhard  shouldn't this be merged with `publish-engine` since they share the same tag?

Reply:
Will be. Highlighted in TODO's here: https://github.com/dagger/dagger/pull/3748#issuecomment-1314074072